packages/roosterjs-editor-api/lib/utils/execCommand.ts

```diff
@@ -32,6 +44,61 @@ export default function execCommand(editor: IEditor, command: DocumentCommand) {
             });
         }
     } else {
-        editor.addUndoSnapshot(formatter, ChangeSource.Format);
+        const handler = (start: NodePosition, end: NodePosition) => {

```

Comment:
same here. Just need to for each selected table cells, call editor.select(td), then go with original code